app/models/user.js

```diff
@@ -30,7 +30,7 @@ export default EmberObject.extend({
     });
 
     return fetch(`${this.runtimeConfig.heliosInternalUrl}${queryString}`, {
-      headers: { 'X-Wikia-Internal-Request': '1' },
+      headers: { 'X-Wikia-Internal-Request': this.runtimeConfig.environment.appName },
```

Comment:
hmm I knew it'll be confusing
we'll have to rethink that...
but

whatever is set in config/environment.js
you 'just' import
import config from '..'';